benchmark/src/main/java/io/scalecube/cluster/gossip/ProtostuffGossipBenchmark.java

```diff
@@ -1,9 +1,9 @@
 package io.scalecube.cluster.gossip;
 
 import io.scalecube.cluster.Member;
-import io.scalecube.transport.Address;
-import io.scalecube.transport.Message;
-import io.scalecube.transport.MessageCodec;
+import io.scalecube.cluster.transport.api.Address;
+import io.scalecube.cluster.transport.api.Message;
+import io.scalecube.cluster.transport.api.MessageCodec;
 

```

Comment:
The transition `io.scalecube.transport` -> `io.scalecube.cluster.transport.api` causes a lot of changed files. Let's split it into separate PR.

Reply:
`io.scalecube.cluster.transport.api` -> `io.scalecube.cluster.transport` ;  it will live in oackage with prefix ` -api `  so it's already clear that this is `api`